Bar Redesign

Tokorizo opened the pull request:
<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->

## What Does This PR Do

- This change will rework the entire bar area, the first change will be the moving of the bar to the kitchen shutters area.

- Due to the moving of the bar I've moved the bartenders backroom east to allow the bartender to access his backroom from the bar while leaving a maint exit as well. 

- The old location of the backroom has been turned into a maint tunnel area with a new door added to the north to allow passage from the dorms to bar maints. 

- I've moved the slot machines, furniture, alarms, APCs, and intercoms to different locations to fit the new design.

 
I've also made some small edits to the kitchen and freezer, 

- I've added a door to the primary hallway in which the chef can use in order to not have always leave through the bar every time. 

- I've made the freeze smaller by moving the entire wall 1 tile to the right to fit for the new backroom.

## Why It's Good For The Game
The main purpose of this design is to put the bar and kitchen next to each other, why? To encourage a more dynamic and cooperative playstyle between the bartender and chef.

To simply put our current bar could be much better than what we have today. Bartender/Chef cooperations in our current design tend to be rather slim or none. Due to the fact that our current bar is secluded from the kitchen. This changes that and allows the Chef to leave food for the Bartender to serve patrons. Patrons will be able to sit at the bar, order drinks, and foods. Not only will this encourage RP and make you feel like your in a diner. It will also add a new dynamic basis for teamwork between the Chef and Bartender.

 To summarize the idea would be fun, encourage RP, and add new dynamic cooperation between the jobs!
## Images of changes
### Current Bar Design
![dreamseeker_G8O5BRFtF6](https://user-images.githubusercontent.com/62493359/78419126-350bc080-7608-11ea-8235-7809c5f23178.png)
### Updated Bar Design
![dreamseeker_6yjQENhYFs](https://user-images.githubusercontent.com/62493359/78419144-5d93ba80-7608-11ea-91a4-6c186688d3a6.png)
### Current Bar Backroom
![dreamseeker_TUfdaakvhf](https://user-images.githubusercontent.com/62493359/78419170-a481b000-7608-11ea-9c2f-8169c8b14ef3.png)
### Update Bar Backroom
![dreamseeker_aU5GdVeF3y](https://user-images.githubusercontent.com/62493359/78419171-af3c4500-7608-11ea-8a1b-d6b6587ad5a5.png)
### Old Bar Backroom Location
![dreamseeker_pqg5Q9bVrs](https://user-images.githubusercontent.com/62493359/78419176-bb280700-7608-11ea-83d5-622d5ca7ed24.png)
### Current Kitchen
![dreamseeker_NJfLRD8faf](https://user-images.githubusercontent.com/62493359/78419186-e6aaf180-7608-11ea-9f8d-cf9a9fc546ae.png)
### Updated Kitchen
![dreamseeker_kbRfFoNMDZ](https://user-images.githubusercontent.com/62493359/78419187-ec083c00-7608-11ea-96c0-c55f641d1040.png)
### Current Freezer
![dreamseeker_EaRgM3wqIi](https://user-images.githubusercontent.com/62493359/78419191-f9bdc180-7608-11ea-8e01-63b24212f9d8.png)
### Updated Freezer
![dreamseeker_CHJs8YN0UR](https://user-images.githubusercontent.com/62493359/78419198-03dfc000-7609-11ea-9f59-5d0a2a7ca1c2.png)
## Changelog
:cl:Tokorizo
tweak: bar redesigned, new door for kitchen, freezer space decreased
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->

Quantum-M commented:
What is the access to the airlock between the bar and the kitchen?

My concern is that the chef/bartender would go into the other's area and start to mess around or do their job.

My suggestion is to either remove the airlock or give the bartender a shutter to that airlock.
The shutter is so the chef is given access to the airlock, and the bartender can keep the way close if both of them are in disagreement to allow the other in their area.

Tokorizo commented:
> What is the access to the airlock between the bar and the kitchen?
> 
> My concern is that the chef/bartender would go into the other's area and start to mess around or do the other's job.
> 
> My suggestion is to either remove the airlock or give the bartender a shutter to that airlock.
> The shutter is so the chef is given access to the airlock, and the bartender can keep the way close if both of them are in disagreement to allow the other in their area.

Very good and fair point sir! I'll get to work on adding the shutter so if the bartender wants to be secluded he can.

Shockpoint commented:
I adore this. I want it. Pls.

Citinited commented:
Is there supposed to be a toolbox just left on the bar?

Spacemanspark commented:
This looks great.

I'd add some shutters to the bar counter itself but that's just me.

Tokorizo commented:
### Update
> The shutter is so the chef is given access to the airlock, and the bartender can keep the way close if both of them are in disagreement to allow the other in their area.

Added Seclusion Shutter so Bartender has option to allow Chef Access to the Bar.
![MwQLMEKDdg](https://user-images.githubusercontent.com/62493359/78456363-cf9bec00-7668-11ea-9796-3f9d641a6693.gif)

> I'd add some shutters to the bar counter itself but that's just me.

Added Privacy Shutters to allow to give an indication that the Bar is closed. It also gives Bartenders more sneaky options if Antaging.
![fykOSiNyvy](https://user-images.githubusercontent.com/62493359/78456437-43d68f80-7669-11ea-9886-4b6974a99524.gif)

> Is there supposed to be a toolbox just left on the bar?

Yes, I left a basic toolbox for all the basic items needed for a Bartender if they ever want to rearrange their bar. (Wrench, Screwdriver, Wire Cutter, Crowbar, Welding Tool.) But if the Bartender whats to go the extra mile to upgrade the bar like; Multitools, Welding Masks, Floor Painters, Etc. He still needs to go to cargo or find one on his own.

> There also seems to be a light switch and entertainment monitor missing from the bar

Monitor re-added.
![dreamseeker_oPLxfQBcEr](https://user-images.githubusercontent.com/62493359/78456550-fb6ba180-7669-11ea-85e3-36e3f4fb1914.png)
As for the light switch, its in the Bartenders backroom. I couldn't really find any space to fit another light switch but the Bartender still has access to the lighting and it's in a location only the Bartender can access if he wants to work on mood lighting
![xRxdCcIW2p](https://user-images.githubusercontent.com/62493359/78456608-50a7b300-766a-11ea-829f-89aaee58ec80.gif)

> You seem to be missing a vent from the kitchen freezer, either that or it's hard to access. Same with a scrubber in the bar back room

They were actually under the Locker/Fridge. But ive moved them into the open so people can see them.
![dreamseeker_oe8IZylHF0](https://user-images.githubusercontent.com/62493359/78456653-7d5bca80-766a-11ea-8c40-4670f592f552.png)
![dreamseeker_oaCHmUHKvQ](https://user-images.githubusercontent.com/62493359/78456655-8056bb00-766a-11ea-8e51-3cb3a7f1d2e4.png)

> If you move the seating area at the bar (and the long bar table) one to the left, you can give the bartender more space without sacrificing much in the way of seating space

Respectfully I think the design is fine, being a Bartender really doesn't require much space to work.
I personally feel this is a good amount of space for the bartender to work in.

### Updated Bar v2
![dreamseeker_3yMh9aAB5H](https://user-images.githubusercontent.com/62493359/78456756-299db100-766b-11ea-88e5-6fec8b2cafd9.png)

Spacemanspark commented:
:+1: love it

Quantum-M commented:
I believe you are missing the bartender's disposal bin
![image](https://user-images.githubusercontent.com/32376559/78459905-5a80e480-7671-11ea-8b79-32dc58485a93.png)
I recommend you put it on the green square, and replace the table in the red square there with a wall (considering that it's already blocked by the chef's disposal bin)

I think the tool box should be moved to the bartender's backroom, otherwise the chef will nab it easily, or a assistant will climb over the counter and steal it. 

On that note, I think the bartender at the minimum just needs a wrench, a screwdriver, and maybe a welder, to descontruct tables and chairs around. I think they should only be given those two/three items instead of a standard tool box.

Fox-McCloud commented:
Not really a fan of making the bartender station a hyper secure area---the chef being able to do so is already kinda an eyebrow raiser, at times---the bartender definitely shouldn't have the same setup---especially since he has a shotgun.

The chef having all the windows he does is a bit better, since you can actually do something about them---but these shutters are effectively a 100% "screw you" no-holds-bar lockdown.

Other issues:
- The bartender being able to lockdown the chef's door. This is just going to invite trouble and also adds unnecessary amounts of security to one of the lowest security places on the station (rightfully so)
- This is going to make it incredibly difficult for the crew to make their own food without tearing things down; as it stands now, you can just hack the door open, bolt it, and the crew can come and go as they please; with this setup, they'll be constantly invading the bartender's space----and he's likely to be less than amiable about letting them do that---even if it's allowed, it's still going to be annoying as heck to put up with people constantly rushing by the equipment that you use.
- This has wayyyy less seating You're sacrificing 4 dining tables chairs, 2 bar stools, and completely lose the 4 lounge seats for....the bartender gaining a crap top of space he doesn't need.
- the flow for the bartender is a lot more labor intensive; most of the current work for the bartender is just walking 1-2 tiles. In this one, you're going to have to constantly be walking multiple tiles up and down just to fill and order. 
- The view that the bartender has doesn't give him a good command over the bar in who he can see; the current design allows him to be able to see the vast majority of seating in the bar; it allows him to serve more than just who's sitting at his table; this design only allows him to see 4 tables chairs, the slot machines, and his regular customers....that's going to decrease interaction.

Overall, not a fan of this at all; it leaves far less seating, axes the lounge, and is a hugely more cramped bar than the previous one; it definitely leaves a lot less room for putting additional knick-knacks or allowing crew based revamps than the current one (diner style, restaurant style, fight center, dance floor, what have you). 

It's way too high security and makes the kitchen way less accessible when there isn't a chef around, makes the bartender's workflow more strenuous---this rework hugely benefits the chef while doing little to improve things for the bartender (other than allowing him to be an occasional jerk and lock the chef out of delivering on the west end of his kitchen). 

I don't think this is going to dramatically increase RP or interaction; it's going to create a lot of new problems while actually decreasing interaction with the bartender, chef, and the rest of the crew----I view the latter as more important than anything that can be gained by increasing interaction between the chef and bartender.

Big thumbs down 👎

Carthusia commented:
> Not really a fan of making the bartender station a hyper secure area---the chef being able to do so is already kinda an eyebrow raiser, at times---the bartender definitely shouldn't have the same setup---especially since he has a shotgun.

**Hyper secure, can you define that please in this context?**
> 
> The chef having all the windows he does is a bit better, since you can actually do something about them---but these shutters are effectively a 100% "screw you" no-holds-bar lockdown.
> 

**Unsure how the bars shutters are different than the kitchens shutters? They are almost less secure due to the windoor aspect, compared to a full airlock, can you please elaborate?**
> Other issues:
> 
> * The bartender being able to lockdown the chef's door. This is just going to invite trouble and also adds unnecessary amounts of security to one of the lowest security places on the station (rightfully so)
**There is a new door to account for this from the kitchen side shutter counter**

> * This is going to make it incredibly difficult for the crew to make their own food without tearing things down; as it stands now, you can just hack the door open, bolt it, and the crew can come and go as they please; with this setup, they'll be constantly invading the bartender's space----and he's likely to be less than amiable about letting them do that---even if it's allowed, it's still going to be annoying as heck to put up with people constantly rushing by the equipment that you use.

**Im not entire sure what you mean by this, the crew shouldnt be storming the kitchen like a medival castle? Do you mean when there is no chef, its harder for people to access the kitchen for food?

If so, there is still the side shutter zone and the side door, and the option to weld past walls to access the kitchen for the criminaly inclined**

> * This has wayyyy less seating You're sacrificing 4 dining tables chairs, 2 bar stools, and completely lose the 4 lounge seats for....the bartender gaining a crap top of space he doesn't need.

**It appears as if the bar only looses 2-4  overall seats , though this is easily modified by custom actions by the individual bartender depending on their tastes**

> * the flow for the bartender is a lot more labor intensive; most of the current work for the bartender is just walking 1-2 tiles. In this one, you're going to have to constantly be walking multiple tiles up and down just to fill and order.

**I dont really feel its as big of an issue as you make it out. 
In the kitchen your running longer distances all the time to do what you need to do, and that doesnt feel like a chore when i'm doing it, and if the bartender wishes, they are free to move their portable/draggable equipment to a new location should they desire to  **

> * The view that the bartender has doesn't give him a good command over the bar in who he can see; the current design allows him to be able to see the vast majority of seating in the bar; it allows him to serve more than just who's sitting at his table; this design only allows him to see 4 tables chairs, the slot machines, and his regular customers....that's going to decrease interaction.

**This i do see your point on, after checking ingame, it seems like a tile view blind spot they do have in compared to the original's 2, but from a different angle, overall they do seem to retain, more or less an acceptable amount of view **

> * You Removed the cake hat. This is important for the chef to make cakes and a super unique item that there's only one of.

**I think this is an oversight more than intentional**
> 
> Overall, not a fan of this at all; it leaves far less seating, axes the lounge, and is a hugely more cramped bar than the previous one; it definitely leaves a lot less room for putting additional knick-knacks or allowing crew based revamps than the current one (diner style, restaurant style, fight center, dance floor, what have you).
> 
> It's way too high security and makes the kitchen way less accessible when there isn't a chef around, makes the bartender's workflow more strenuous---this rework hugely benefits the chef while doing little to improve things for the bartender (other than allowing him to be an occasional jerk and lock the chef out of delivering on the west end of his kitchen).
> 
> I don't think this is going to dramatically increase RP or interaction; it's going to create a lot of new problems while actually decreasing interaction with the bartender, chef, and the rest of the crew----I view the latter as more important than anything that can be gained by increasing interaction between the chef and bartender.
> 
> Big thumbs down 👎

**Im in the opposite of view, the bar retains a large enough space for boxing arenas, diners, cafes, restaurants and other mixtures and people are still able to swap out the layout as they see fit. 

The old bar has about 96 tiles for custom usage, while the new has 88, thats only a 8 tile drop for an overall better bar look and experience

Overall the design to me, i an improvement over the old one and is a nice default bar to save people time redesigning it, enable more RP and interactions while still allowing for people to swap things out as they wish**

Tokorizo commented:
### Update
> I believe you are missing the bartender's disposal bin
> ![image](https://user-images.githubusercontent.com/32376559/78459905-5a80e480-7671-11ea-8b79-32dc58485a93.png)

Fixed it good sir, I've also removed the toolbox and added a wrench, crowbar, and screwdriver in the pile of Bartenders things in the backroom. Anything else must be earned from other sources.
![dreamseeker_kj6ATTEXUd](https://user-images.githubusercontent.com/62493359/78460939-d9364b80-768a-11ea-8877-1e522257e65e.png)







> Not really a fan of making the bartender station a hyper secure area---the chef being able to do so is already kinda an eyebrow raiser, at times---the bartender definitely shouldn't have the same setup---especially since he has a shotgun.

I respectfully don't belive the Bar due to adding of shutters is a "Hyper Secure Area" if I was really going for something along those lines I would have added Reinforced walls, Even more shutters along the Bar, And less glass for visibility. 5 shutters really shouldn't cause that many issues.

> The chef having all the windows he does is a bit better, since you can actually do something about them---but these shutters are effectively a 100% "screw you" no-holds-bar lockdown.

I don't really understand the meaning of "no holds" but I'm going to assume you mean that the chef can close the windows and deny the bartender food. If that's the case then its something that simply happens, some chefs/bartenders prefer to work alone and that perfectly fine.


> * The bartender being able to lockdown the chef's door. This is just going to invite trouble and also adds unnecessary amounts of security to one of the lowest security places on the station (rightfully so)

Please clarify by what you mean "invite trouble". This doesnt really tell me anything to improve or keep in mind, its just a vague statement that adding this is "bad". Also for the security concern please look at the next statement.

> * This is going to make it incredibly difficult for the crew to make their own food without tearing things down; as it stands now, you can just hack the door open, bolt it, and the crew can come and go as they please; with this setup, they'll be constantly invading the bartender's space----and he's likely to be less than amiable about letting them do that---even if it's allowed, it's still going to be annoying as heck to put up with people constantly rushing by the equipment that you use.
...
![hmTsxo8gWH](https://user-images.githubusercontent.com/62493359/78461594-17823980-7690-11ea-8eba-d1ec4fc597ab.png)

> * the flow for the bartender is a lot more labor intensive; most of the current work for the bartender is just walking 1-2 tiles. In this one, you're going to have to constantly be walking multiple tiles up and down just to fill and order.

This personally seems like a bit of a nitpick and let me explain why, you're saying that the bartender actually has to move and serve customers almost as if this were a bad gameplay design when in effect it makes the Bartender job more stuff to do, instead of walking 3 tiles in total to serve drinks your now walking either 4 or 5 tiles to serve drinks AND food, 1 second has been lost from this process and I respectfully don't agree with this opinion.

> * This has wayyyy less seating You're sacrificing 4 dining tables chairs, 2 bar stools, and completely lose the 4 lounge seats for....the bartender gaining a crap top of space he doesn't need.

So let's run some numbers. There are 16 wooden chairs, 5 barstools, a 3 regular stools. Our current design has 20 wooden chairs, 7 barstools, and 4 lounge chairs. In total there is a difference in the seating of about 7 seats. Now the first thing you need to keep in mind is that your implying that not having 31 seats compared to 24 is a drastic change, 2 of those missing barstools were just stools for the slot machines. You could only ever serve 5 people at the bar in the old design and this design as well. The lounge chairs also become incredibly irrelevant when a Bartender decides to upgrade the seating of the bar with all Comfy chairs, also the lounge was not good in my opinion. too small, felt forced in, and honestly, you can just add your own comfy chairs to make an ACTUAL lounge. Getting ALL 31 of those seats filled by someone is impossible unless there was an actual special event in the bar. Most time players will come in, sit and leave after a certain period of time with new players coming in to replaces their seats. and never has the bar reached a "max" capacity with anything close to 31 seats. Also, you have metal and tools in the back if making more seating is that much of an issue.

> * The view that the bartender has doesn't give him a good command over the bar in who he can see; the current design allows him to be able to see the vast majority of seating in the bar; it allows him to serve more than just who's sitting at his table; this design only allows him to see 4 tables chairs, the slot machines, and his regular customers....that's going to decrease interaction.

I actually can understand this point of view and agree with it in a way, but one can argue that having a blindspot in the bar encourages the bartender to walk around, clean the bar, fix anything, and actually inspect the bar. Also having blindspots doesn't inherently have to be a bad thing like I said earlier gives an incentive to walk around and see if everything is in order. Also, I disagree with the decrease in interaction, whenever you played bartender your focus was on the 5 people sitting at the bar asking for drinks. If nobody was at the bar, you would either sit at the bar and wait for people to sit down to take their drinks, or seek out people in the bar and ask them what they want. You can still do that but now you can ask them if they want food, drinks, or both.

You make some valid points but I respectfully disagree with most of them. I believe the current bar will be much better than our current one by a long shot.

Quantum-M commented:
> Not really a fan of making the bartender station a hyper secure area---the chef being able to do so is already kinda an eyebrow raiser, at times---the bartender definitely shouldn't have the same setup---especially since he has a shotgun.

I would say that it is slightly more secure, but there are at least three different points that people can break in/enter into the bar, that being the side windoor, the chef's door, and the backroom door.

> The bartender being able to lockdown the chef's door. This is just going to invite trouble and also adds unnecessary amounts of security to one of the lowest security places on the station (rightfully so)

There's a side door for the chef leading to the main hallway, the shutter to the bar was a recommendation on my part in concerns of the chef/bartender trespassing into the other's area when it was unwarranted, that way the chef could keep control on their airlock, and the bartender could keep them out with the small shutter.

> This is going to make it incredibly difficult for the crew to make their own food without tearing things down; as it stands now, you can just hack the door open, bolt it, and the crew can come and go as they please; with this setup, they'll be constantly invading the bartender's space----and he's likely to be less than amiable about letting them do that---even if it's allowed, it's still going to be annoying as heck to put up with people constantly rushing by the equipment that you use.

There's a side door to the main hallway. And in eithercase, I don't think we should even encourage people to break into places if the chef is not present at the moment/non-existent. There tends to be quite a lot of high pop and a HoP to deal with that, and there seems to be a tendency for people to do other people's job while not having said job at the moment.

> This has wayyyy less seating You're sacrificing 4 dining tables chairs, 2 bar stools, and completely lose the 4 lounge seats for....the bartender gaining a crap top of space he doesn't need.

I have barely seen the whole seating area being used at all in the current, the most I recall recently is one small table and the whole bar seating.

As for the lounge area, I am assuming you are referring to the comfy brown chairs. I have seen it being largely used for putting SSDs than for any extended interaction.

> The view that the bartender has doesn't give him a good command over the bar in who he can see; the current design allows him to be able to see the vast majority of seating in the bar; it allows him to serve more than just who's sitting at his table; this design only allows him to see 4 tables chairs, the slot machines, and his regular customers....that's going to decrease interaction.

Most patrons I seen usually get served at the bartender's counter, and eitherway, the bartender not having full view of the whole bar at least gives some antags to do some take down/kidnap their target without a mob and the bartender going/shooting after them.

Pixelzilla commented:
I just don't like the overall flow of this design. It has a really bad floor plan the older bar design actually was pretty nice and visually appealing but this one seems odd with the bar situated on the immediate right-hand side isn't very inviting since the first visual you'll probably get is a long empty-ish space and a really busy area to the right.

With the current bar you can walk in and go immediately to the bar or deviate to a table if you'd like. Your path is unobstructed, the room feels large yet not empty but it also doesn't feel cluttered/tightly packed together.

With the suggested bar your path is almost immediately obstructed by a table and while the other door does offer a straight shot to the bar it does end with having to turn to the right. If the bar is busy and those tables are being used you're going to run into traffic issues that the current bar layout doesn't have.

The room may be "longer" but it doesn't leave a lot of options for revamping the bar easily. If you want to do a unique design the current bar is ideal for that and you have a lot of freedom that doesn't require you to put in a ton of effort before reaping the rewards. Dance floor? Sure. Booth seating? Absolutely. Unfortunately you don't have a lot of options for alternative table placements with the proposed changes.

It also seems odd that the bar would be right up against the kitchen in the first place. Who has access to the door between the bartender's room/kitchen now? Is it bartender? Is it the chef? Is it both? Why do either need access to the other's space and why should either of them being tromping into someone else's area and thus trespassing?

I understand that the goal here seems to be to force cooperation between chef/bartender but it doesn't allow for the opposite playstyle. I feel it is far more likely that people who do want to work together will revamp the bar together in a setup they prefer vs two people who don't want to cooperate together somehow cooperating enough to revert the bar back to a different layout.

- Less privacy for both chef/bartender to do any antagging in their respective back rooms.
- Chef's freezer is giving up space, 1 meat hook and even obstructs easy access to the freezer crate.
- Layout will cause traffic issues, awkward flow.
- Forces chef/bar to be in close proximity to each other - doesn't really allow for differing playstyles/preference.

Tokorizo commented:
> I just don't like the overall flow of this design. It has a really bad floor plan the older bar design actually was pretty nice and visually appealing but this one seems odd with the bar situated on the immediate right-hand side isn't very inviting since the first visual you'll probably get is a long empty-ish space and a really busy area to the right.

The overall flow of design I something new and different, obviously this is something that will take time once the map design is run. The floor plan is something that really is just limitations of the area that can be worked with without taking MORE of the station for the bar and even then I feel like ive done a pretty good job at making sure there is enough seating and passage, navigation really shouldn't be an issue because there are multiple options of movement in the bar and even WHEN the bar is full my design or our current design, its always crowded and people are usually pushing each other around. Visual appealing is just opinion-based, but I'm sure more people prefer mine than the current design. Also, I disagree with that just because the Bar on the right-hand side isn't inviting, when you walk through the doors you will immediately see the bartender and bar stools. A pretty clear indication of where to sit and get drinks and food. 

> With the current bar you can walk in and go immediately to the bar or deviate to a table if you'd like. Your path is unobstructed, the room feels large yet not empty but it also doesn't feel cluttered/tightly packed together.

You can still do that in my design, left hand for seating, right hand for bar or seating as well. Multiple options. The only difference is the bar is to the right instead of being the left corner and the basic layout difference. (Wider vs Taller) 

> With the suggested bar your path is almost immediately obstructed by a table and while the other door does offer a straight shot to the bar it does end with having to turn to the right. If the bar is busy and those tables are being used you're going to run into traffic issues that the current bar layout doesn't have.

Again the other bar follows the same problems as well if the bar is busy/packed you also run into traffic problems, this design isn't to fix the traffic problem the bar has all the time. It's just to move the bar next to the kitchen for a different reason.  Even in the other design if we were to say all seats were filled with players there would still only be 2 actual routes to get to the bar. The same applies for my design as well. Again this redesign is not to fix traffic problems.

> The room may be "longer" but it doesn't leave a lot of options for revamping the bar easily. If you want to do a unique design the current bar is ideal for that and you have a lot of freedom that doesn't require you to put in a ton of effort before reaping the rewards. Dance floor? Sure. Booth seating? Absolutely. Unfortunately you don't have a lot of options for alternative table placements with the proposed changes.

In a previous post (if you bothered to read it), we actually ran a difference on how much tiles were available for use compared to our current design and the remodel design. The current bar has 96 while New bar has 88, a drop of 8 tiles. If you somehow cannot make a boxing ring, stage, dance floor, etc with 8 fewer tiles of space in an already customizable bar then I really can't fix that issue. You can still build whatever extra thing you want, 8 tiles difference is not a massive loss for customization. And as for seating, this problem will go away once people actually get used to the new design and find out how they want to customize/work on their own layouts, I would argue this new design will make more people create new design we haven't seen yet. Just because of how new and different the layout is.

> It also seems odd that the bar would be right up against the kitchen in the first place. Who has access to the door between the bartender's room/kitchen now? Is it bartender? Is it the chef? Is it both? Why do either need access to the other's space and why should either of them being tromping into someone else's area and thus trespassing?

Windoor to the bar is only Bartender Access if the bartender does not want the Chef in the bar I've added a shutter button to keep the chef out. The Chef still has access to the door. If he wants to come in but cannot unless the shutter is unlocked by the Bartender. This actually bring a pretty good point and ill prob work a new design for isolation if neither the Bartender or Chef wants to cooperate, more on that later. Now just realizing that the Chef has access to the windoors and the Bartender cannot do anything if the Chef decides to throw stuff into the bar. 

> I understand that the goal here seems to be to force cooperation between chef/bartender but it doesn't allow for the opposite playstyle. I feel it is far more likely that people who do want to work together will revamp the bar together in a setup they prefer vs two people who don't want to cooperate together somehow cooperating enough to revert the bar back to a different layout.

Again, ill get an update for a design that is optional if neither party wants to cooperate, which I think would be pretty rare because there really is no reason for the Chef or Bartender to hate or dislike working with each other but again sometimes this situation does happen so stay tuned. Also, I disagree with the whole Revamp thing, I've been playing this game for a good while and I have yet to see anyone make bar designs similar to mine, the reason why is it takes a lot of work, time, and energy just to move everything around and that is a big deterrent to cooperation designs like mine.

> * Less privacy for both chef/bartender to do any antagging in their respective back rooms

Disagree, the Chef and Bartender still have their backrooms to do their antaging shenanigans, The Bar backroom is the same size as the original bar backroom and the chef still has the important antag tools, (Gibber/Locker) only at the loss of a tiny space. id argue that there is more potential for antag opportunity due to the new shutters and blind spot in the bar, and the connecting Bar and Freezer design.

> * Chef's freezer is giving up space, 1 meat hook and even obstructs easy access to the freezer crate.

4 tiles of space from a room the chef spends 10% of his time in only to butcher and hold corpses in one locker. Also, loss of 1 meathook is not much of a game-changer, there rebuildable and most chefs use one at a time for obvious purposes, again not a massive blow. And for the freezer crate, ever heard of just moving the box? Ctrl + Click is really easy and not difficult.

> * Layout will cause traffic issues, awkward flow.

Again, our current bar tends to suffer the same problem, and my goal isn't to fix the always present traffic problems. Its to get a new better design that encourages teamwork and diner-like experience, like how the bar should be.

> * Forces chef/bar to be in close proximity to each other - doesn't really allow for differing playstyles/preference.

Again, you act like the Chef/Bar is some sort of rivalry when in fact they both have the same goal, serve the crew and do their respected jobs. Also please list any other playstyles which do not fall under Not Cooperating and Cooperating, please.

> * Also want to point out that you can always coordinate with the service headsets. I do it all the time.

Yes, but now the potential for cooperation is extremely better and the layout is set, Similar to how the Chef and Botanist work, it will be the Bartender and Chef in this design. And unlike botany/chef the potential for Bartending does not depend on his partner in order to serve drinks. They can still do their jobs to their full potential, even without the other present.

Citinited commented:
I agree with Fox's comments about workflow for the bartender. Say the chef wants to exit the kitchen and the bartender is in the way. He can either push past the bartender (which will be super awkward if either is not on help intent), wait for the bartender to finish (this might take a while, depending on if the bartender is making a complex drink or AFK), or go through another entrance (super not ideal). 

TBH the chef no longer needs a super long table with these changes. Given he/she only really needs a few tables for passing food to the chef, this is just wasted table space. You could install some windows or something and plonk the drinks machines there instead.

> instead of walking 3 tiles in total to serve drinks your now walking either 4 or 5 tiles to serve drinks AND food, 1 second has been lost from this process and I respectfully don't agree with this opinion. 

The old layout only required the bartender to move 1 tile maximum to be able to pass anyone a drink. This is 5 to be able to pass everyone a drink. It's obviously a small difference, but small differences do add up

Regarding the long 2 by 4 table, why not swap it with the tables by the air alarm? This would make the middle of the bar feel a lot more open

> I've also removed the toolbox and added a wrench, crowbar, and screwdriver in the pile of Bartenders things in the backroom. Anything else must be earned from other sources.

Why does the bartender need these tools at roundstart?

Spacemanspark commented:
Some bartenders like to customize their bar.

Carthusia commented:
> I agree with Fox's comments about workflow for the bartender. Say the chef wants to exit the kitchen and the bartender is in the way. He can either push past the bartender (which will be super awkward if either is not on help intent), wait for the bartender to finish (this might take a while, depending on if the bartender is making a complex drink), or go through another entrance (super not ideal).

**As a hardcore chef player, exiting through the new side door provided to the main hallway, is not an issue in the slightest in my eyes. It is an option depending on the scenario, and it is a welcome one**
> 
> TBH the chef no longer needs a super long table with these changes. Given he/she only really needs a few tables for passing food to the bartender for serving, this is just wasted table space. You could install some windows or something and plonk the drinks machines there instead.

**I Highly disagree, when I operate as the chef, I expand my tablespace almost 4 fold to be able to properly organize my tools, food, prep zone and to create a buffet visible to players to request food from to avoid Food poisoning, regular poisoning, ants and pre-munched food. More table space, is better when you are operating as a chef.**
> 
> > instead of walking 3 tiles in total to serve drinks your now walking either 4 or 5 tiles to serve drinks AND food, 1 second has been lost from this process and I respectfully don't agree with this opinion.
> 
> The old layout only required the bartender to move 1 tile maximum to be able to pass anyone sat at the bar a drink. This is 5 to be able to pass everyone a drink. It's obviously a small difference, but small differences do add up

**The bartender, should they wish, is capable of simply unwrenching their drink machines, and moving them directly behind them by pulling them. This is more of a preference issue than a practical issue.**
> 
> Regarding the long 2 by 4 table, why not swap it with the tables by the air alarm? This would make the middle of the bar feel a lot more open

**I could see instead of any giant table, simply having small tables as appealing, though that's more personal preference. The new design seems to be an amalgamation of different ideas to combine them into a default bar that appeals to a larger group of players than the old design at the start, but still allow them to customize it as they see fit.**
> 
> > I've also removed the toolbox and added a wrench, crowbar, and screwdriver in the pile of Bartenders things in the backroom. Anything else must be earned from other sources.
> 
> Why does the bartender need these tools at roundstart?

**As Spark noted, some bartenders LOVE to redesign their bar entirely(<------Me), this is the reason they start with 50 metal and glass in their back room, some simple construction tools enables them to quickly begin this process at round start should they desire to**

Citinited commented:
> As a hardcore chef player, exiting through the new side door provided to the main hallway, is not an issue in the slightest in my eyes. It is an option depending on the scenario, and it is a welcome one

The door is nice, but I would not want to use it if there's a crowd of greytide milling round the food. IMO it's a bit awkward but point taken regarding the new door

> The bartender, should they wish, is capable of simply unwrenching their drink machines, and moving them directly behind them by pulling them onto the table. This is more of a preference issue than a practical issue.

Why should bartenders have to redesign their workspace if it was badly designed in the first place?

Having to walk 10 turfs versus walking 2 turfs every time you want to simply get a patron a drink is not "more of a preference issue than a practical issue"

Also I don't believe you can pull a drinks dispenser from the floor onto a table without someone else to pull it on top of you, unless that was changed recently.

> I Highly disagree, when I operate as the chef, I expand my tablespace almost 4 fold to be able to properly organize my tools, food, prep zone and to create a buffet visible to players to request food from to avoid Food poisoning, regular poisoning, ants and pre-munched food. More table space, is better when you are operating as a chef.

Sure, but this inaccessible (for normal crew) table is 2 turfs longer than the publicly accessible table in starboard primary.

There's a balance to be struck between the chef making beautifully prepared displays of food that take up lots of room and using that space for better workflow, for example placing the drinks dispensers near where the bartender will be serving his patrons.


> Some bartenders like to customize their bar.

Then they can go to tool storage, cargo, or ask an engineer to assist, same as everyone else

Carthusia commented:
> The door is nice, but I would not want to use it if there's a crowd of greytide milling round the food. IMO it's a bit awkward but point taken regarding the new door

**Heh, To be fair, the frothing greytide masses demanding food at the door, would be the same with the side door or the original bar door**
> 


> Also I don't believe you can pull a drinks dispenser from the floor onto a table without someone else to pull it on top of you, unless that was changed recently.
**From what I recall, you simply have to click the floor below the table to pull a small object like that onto it** 

> 
> Sure, but this inaccessible (for normal crew) table is 2 turfs longer than the publicly accessible table in starboard primary. There's a balance to be struck between the chef making beautifully prepared displays of food that take up lots of room and using that space for better workflow, for example placing the drinks dispensers near where the bartender will be serving his patrons to reduce the amount of walking required.

**Its more a matter of personal preference, the new design of the bar looks to be a combination of concepts and ideas to enable more RP, interaction than before, its a much better stater bar than the previous one in my eyes, and should a player wish, they are free to customize it as they see fit with the provided materials. 

The same goes with the kitchen, each player is allowed to customize it to fit their playstyle and work style, it has a default starter which is meant to be conducive to working it properly **

> 
> > Some bartenders like to customize their bar.
> 
> Then they can go to tool storage or cargo for tools, or ask an engineer to assist, same as everyone else
**The materials they can use for customisation have been in their bar for at least a year, simply adding in the tools to disassemble the tables is a quality of life thing.
Why give them the materials they need, but not the basic tools?**

Overall, I still see more positives than negatives with the new layout compared to the old, the bar/kitchen are Social jobs, whether a player plays it that way is up to them, but they are in fact social positions on the station, and allowing both of them to work closer together increases social interactions of both jobs with each other and the crew.

Tokorizo commented:
> I agree with Fox's comments about workflow for the bartender. Say the chef wants to exit the kitchen and the bartender is in the way. He can either push past the bartender (which will be super awkward if either is not on help intent), wait for the bartender to finish (this might take a while, depending on if the bartender is making a complex drink), or go through another entrance (super not ideal).

The new door was added in the south of the kitchen, and Bartender has full control over his workplace with the new shutter lock-off button I've added. The chef must be allowed access into the bar via permission. 

> The old layout only required the bartender to move 1 tile maximum to be able to pass anyone sat at the bar a drink. This is 5 to be able to pass everyone a drink. It's obviously a small difference, but small differences do add up

A little trick I like to do to reduce walk times is standing at the bar, wait for a patron, get their order. Walk to the machines, which I would like to add are all 3 connected to each other so you can access all the machines at once. Prepare the drink and simply just slide it down the bar, or throwing is a more accurate term. A little trick good bartenders will use to reduce walking speeds and also increase drink production. Heres an example.
![xZJ2ILnGV2](https://user-images.githubusercontent.com/62493359/78695194-6237c780-78c3-11ea-8f9d-0cb097825d79.gif)

> Regarding the long 2 by 4 table, why not swap it with the tables by the air alarm? This would make the middle of the bar feel a lot more open

Not a big fan of having small tables everywhere but I do hear the criticism of space. So ill be simply reducing the table size to a 1x4 instead of 2x4 to help in the space issue.
![dreamseeker_Pr39sNeJat](https://user-images.githubusercontent.com/62493359/78695661-002b9200-78c4-11ea-850a-5114e5cec440.png)

> The door is nice, but I would not want to use it if there's a crowd of greytide milling round the food. IMO it's a bit awkward but point taken regarding the new door

Its always like that, and if you really don't wanna leave through the front door, the freeze back door is still available. Crowding for food has always happened, even with the current design. I cant fix gray tide mobbing for food, only a chef who actually does their job and serves food can please the hoards. Also, CQC for any trouble makers is usually the answer. (Not Lethal just simply restraining them is what I mean.)

> Then they can go to tool storage or cargo for tools, or ask an engineer to assist, same as everyone else

The tools are just simple quality of life changes to allow customization, if the Bartender wants to do a massive remodel or wants to hack the drink machines, they still require cargo/engineering for supplies and tools.

Also, the cake hat is back!

MRgaty commented:
I dont like the booze dispensers being there, it forces you to be in the middle of the chef's way, if the bartender happens to be in disarm intent or such it will make entering the kitchen that way annoying.

Carthusia commented:
> I dont like the booze dispensers being there, it forces you to be in the middle of the chef's way, if the bartender happens to be in disarm intent or such it will make entering the kitchen that way annoying.

This has been pointed out multiple times, and multiple times it has been pointed out:

The kitchen has a new side door for easy access to the kitchen from the main hall and the bartender has the ability to lock down the bar door to the kitchen if they so desire.

This is all visible in the above posts.

There will be no interruptions to the bartender.

Tokorizo commented:
> I dont like the booze dispensers being there, it forces you to be in the middle of the chef's way, if the bartender happens to be in disarm intent or such it will make entering the kitchen that way annoying.

I've already addressed this point multiple times so allow me to clarify so there are no longer any misunderstandings, I've added a new door to the kitchen so the chef can leave the kitchen without going through the bar. The Bar is locked off to the chef unless the bartender allows them in. The Bar windoor is Bartender only so the only place a chef has to enter is their backdoor freezer or the new door I've added if they want to enter the kitchen. This way the bartender is not disturbed in his workplace every time the chef comes and goes.

Aquilar commented:
this bar looks like some of the pubs I've been in.

Tokorizo commented:
Should be done, ill @ you whenever I'm available to TM it. If anything is wrong just tell me so I can fix it.

Tokorizo commented:
a design that was made on the first TM

![0143797a1a4261d026d1d84d67e6bd55](https://user-images.githubusercontent.com/62493359/81137985-101bad80-8f26-11ea-9d54-4bbac857c56e.png)

sadly it was a cling round... either way, people seemed to like it.

Carthusia commented:
Overall people seemed to like the bar layout, though yea, it just happened to be a super short sling round and it didn't get a whole lot of test time.

Perhaps another TM soon to get a bit more ingame usage to see how people design it, use it and what not.

PhantasmicDream commented:
While I do enjoy seeing map reworks for the station I personally am not a fan of this change. 

I agree with points like how this is a weird flow, and my bias opinion of hating bar counters being close to the front entrance way.  I get what you're trying to go for but I think you're missing the mark.  Simply moving the bar counter in front kitchen counter seems counter productive. 

You may have to rework the whole bar plus kitchen and possibly botany to get it flowing better. Honestly I suggest just jumping into other servers (or checking out different maps) to see how their bar/kitchen lay out is.  Because I feel like you're trying to keep it's original shape too much, and it probably makes it harder to lay things out.   

An example could be sliding the bar counter back to the back and over closer to the kitchen and condensing the size of the kitchen a bit because it doesn't necessarily need to be a big as it is. and then add a dining area in front of it. 
![](https://cdn.discordapp.com/attachments/151095049982181377/708178958025359440/Untitled_Artwork.jpg)

variableundefined commented:
Closing due to majority maintainer and head objections.
